src/provider.rs

```diff
@@ -1020,8 +1040,30 @@ impl FromStr for Ticket {
     }
 }
 
+/// Create a [`quinn::ServerConfig`] with the given keypair and limits.
```

Comment:
why is this public now?

Reply:
I don't know, I think @rklaehn wrote this.  I only moved it since it was below the tests module for some reason (so i missed there were already tests and then the compiler complained that i can't have two tests modules)